src/calendars/poland.js

```diff
@@ -335,17 +335,10 @@ let dates = year => {
       }
     },
     {
-      "key": "saintHenryBishopAndMartyr",
+      "key": "saintCamillusDeLellisPriestOrSaintHenryBishopAndMartyr",
```

Comment:
@jarosz, I don’t think it is celebrated like this in Poland. If the Wiki is correct, you don’t even celebrate St Henry. Pleasd, verify this in the official sources.

Reply:
Polish calendar got changed in 2018, I'll have to update the wiki according to https://opoka.org.pl/biblioteka/T/TA/TAL/kkbids/Kalendarz_diecezji_polskich_2018.pdf and review whole poland.js

Reply:
Okay then, you know it better. :smiley: